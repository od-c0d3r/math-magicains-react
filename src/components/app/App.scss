#App {
  font-family: 'Oxygen', sans-serif;
}
