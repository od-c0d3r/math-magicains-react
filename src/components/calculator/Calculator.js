import './Calculator.scss';
import React from 'react';
import Screen from '../screen/Screen';
import Keyboard from '../keyboard/Keyboard';

class Calculator extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div id="calContainer">
        <Screen />
        <Keyboard />
      </div>
    );
  }
}

export default Calculator;
