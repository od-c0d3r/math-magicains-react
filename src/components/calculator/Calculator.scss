#calContainer {
  display: grid;
  grid-template-columns: auto;
  border-radius: 10px;
  box-shadow: rgba(0, 0, 0, 0.3) 0 19px 38px, rgba(0, 0, 0, 0.22) 0 15px 12px;
}
