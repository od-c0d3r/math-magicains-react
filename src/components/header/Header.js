import './Header.scss';
import Navbar from '../navbar/Navbar';

const Header = () => (
  <header id="appHeader">
    <span id="logo">
      LIFE
      <sub>
        OF
      </sub>
      PI
    </span>
    <Navbar />
  </header>
);

export default Header;
