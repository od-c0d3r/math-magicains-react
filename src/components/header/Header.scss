#appHeader {
  display: flex;
  justify-content: space-between;
  align-items: center;
  box-shadow: rgba(0, 0, 0, 0.24) 0 3px 8px;
  height: 120px;

  #logo {
    text-shadow: 0 4px 3px rgba(0, 0, 0, 0.4), 0 8px 13px rgba(0, 0, 0, 0.1), 0 18px 23px rgba(0, 0, 0, 0.1);
    padding-left: 20px;
    font-family: "Gideon Roman", cursive;
    font-size: 5vw;
  }
}
