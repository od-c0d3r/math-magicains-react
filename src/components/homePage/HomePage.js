import './HomePage.scss';

const HomePage = () => (
  <div id="homePage">
    <p>
      if you were to play a word association game with a
      school-aged child, you&apos;d be pretty unlikely to get a response of
      &apos;cool&apos; when you asked them what word first came into their
      mind when you said &apos;math&apos;.
      Despite what some people may tell you, math is far from dull.
      On the contrary, there are plenty of fun and strange
      math-related facts out there that will fascinate children of all ages.
      To prove this,
      we&apos;ve compiled a list of 20 cool facts about math which we encourage you
      to share with the children in your life.
    </p>

    <p>
      1. The word &quot;hundred&quot; comes from the old Norse term, &quot;hundrath&quot;,
      which actually means 120 and not 100.
    </p>
    <p>
      2. In a room of 23 people there&apos;s a 50% chance that two people have the same birthday.
    </p>

    <p>
      3. Most mathematical symbols weren&apos;t invented until the 16th century.
      Before that, equations were written in words.
    </p>
    <p>
      4. &quot;Forty&quot; is the only number that is spelt
      with letters arranged in alphabetical order.
    </p>

    <p>
      5. Conversely, &quot;one&quot; is the only number that is spelt
      with letters arranged in descending order.
    </p>
    <p>
      6. From 0 to 1000, the only number that has
      the letter &quot;a&quot; in it is &quot;one thousand&quot;.
    </p>
    <p>
      7. &apos;Four&apos; is the only number in the English language that is spelt
      with the same number of letters as the number itself.
    </p>
    <p>
      8. Every odd number has an &quot;e&quot; in it.
    </p>
    <p>
      9.Over 2000 years ago, Eratosthenes estimated the Earth&apos;s circumference
      using math, without ever leaving Egypt, and he was accurate to within 2%
    </p>
    <p>
      10. Markings on animal bones indicate that humans have been doing math since around 30,000BC.
    </p>
    <p>
      11. &quot;Eleven plus two&quot; is an anagram of &quot;twelve plus
      one&quot; which is pretty fitting as the answer to both equations is 13.
    </p>
    <p>
      12. Also, there are 13 letters in both
      &quot;eleven plus two&quot; and &quot;twelve plus one&quot;.
    </p>
    <p>
      13. Zero is not represented in Roman numerals.
    </p>
    <p>
      14. The word &quot;mathematics&quot; only appears in one Shakespearean play,
      &quot;The Taming of the Shrew&quot;.
    </p>
    <p>
      15. -40 °C is equal to -40 °F.
    </p>
    <p>
      16. In France, a pie chart is sometimes referred to as a &quot;camembert&quot;.
    </p>
    <p>
      17. The symbol for division (i.e.÷) is called an obelus.
    </p>
    <p>
      18. 2 and 5 are the only prime numbers that end in 2 or 5.
    </p>
    <p>
      19. A &apos;jiffy&apos; is an actual unit of time. It means 1/100th of a second.
    </p>
    <p>
      20. If you shuffle a deck of cards properly, it&apos;s more than likely that
      the exact order of the cards you get has never been seen before in the whole
      history of the universe.
    </p>
  </div>
);

export default HomePage;
