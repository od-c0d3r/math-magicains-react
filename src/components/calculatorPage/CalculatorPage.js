import './CalculatorPage.scss';
import Calculator from '../calculator/Calculator';
import CalContent from '../calContent/calContent';

const CalculatorPage = () => (
  <div id="calPage">
    <CalContent />
    <Calculator />
  </div>
);

export default CalculatorPage;
