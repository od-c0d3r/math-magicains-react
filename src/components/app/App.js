import './App.scss';
import 'normalize.css';
import React from 'react';
import Header from '../header/Header';
import CalculatorPage from '../calculatorPage/calculatorPage';

const App = () => (
  <div id="App" style={{ height: window.innerHeight }}>
    <Header />
    <CalculatorPage />
  </div>
);

export default App;
