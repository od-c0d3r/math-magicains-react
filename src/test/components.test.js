import TestRenderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import App from '../components/app/App';
import HomePage from '../components/homePage/HomePage';
import CalculatorPage from '../components/calculatorPage/CalculatorPage';
import QuotePage from '../components/quotePage/QuotePage';
import CalContent from '../components/calContent/calContent';
import Header from '../components/header/Header';
import Navbar from '../components/navbar/Navbar';
import Keyboard from '../components/keyboard/Keyboard';
import Screen from '../components/screen/Screen';
import Key from '../components/key/Key';
import Calculator from '../components/calculator/Calculator';

describe('Components Snapshots', () => {
  test('renders correctly App', () => {
    const tree = TestRenderer.create(<BrowserRouter><App /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly Navbar', () => {
    const tree = TestRenderer.create(<BrowserRouter><Navbar /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly Keyboard', () => {
    const keyNums = ['AC', '+/-', '%', 'รท', '7', '8', '9', 'x', '4', '5', '6', '-', '1', '2', '3', '+', '0', '.', '='];

    const tree = TestRenderer.create(
      <BrowserRouter>
        <Keyboard keyNums={keyNums} />
      </BrowserRouter>,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly Screen', () => {
    const tree = TestRenderer.create(<BrowserRouter><Screen screen="0" /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly Key', () => {
    const tree = TestRenderer.create(<BrowserRouter><Key /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly CalContent', () => {
    const tree = TestRenderer.create(<CalContent />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly Calculator', () => {
    const tree = TestRenderer.create(<Calculator />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly Header', () => {
    const tree = TestRenderer.create(<BrowserRouter><Header /></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly HomePage', () => {
    const tree = TestRenderer.create(<HomePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly CalculatorPage', () => {
    const tree = TestRenderer.create(<CalculatorPage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  test('renders correctly QuotePage', () => {
    const tree = TestRenderer.create(<QuotePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
