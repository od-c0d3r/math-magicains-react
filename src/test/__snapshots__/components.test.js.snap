// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Snapshots renders correctly App 1`] = `
<div
  id="App"
  style={
    Object {
      "height": 768,
    }
  }
>
  <header
    id="appHeader"
  >
    <span
      id="logo"
    >
      LIFE
      <sub>
        OF
      </sub>
      PI
    </span>
    <nav
      id="navbar"
    >
      <ul>
        <li>
          <a
            activeclassname="active"
            className=""
            href="/home"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            activeclassname="active"
            className=""
            href="/calc"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            activeclassname="active"
            className=""
            href="/quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </header>
  <div
    id="homePage"
  >
    <p>
      if you were to play a word association game with a school-aged child, you'd be pretty unlikely to get a response of 'cool' when you asked them what word first came into their mind when you said 'math'. Despite what some people may tell you, math is far from dull. On the contrary, there are plenty of fun and strange math-related facts out there that will fascinate children of all ages. To prove this, we've compiled a list of 20 cool facts about math which we encourage you to share with the children in your life.
    </p>
    <p>
      1. The word "hundred" comes from the old Norse term, "hundrath", which actually means 120 and not 100.
    </p>
    <p>
      2. In a room of 23 people there's a 50% chance that two people have the same birthday.
    </p>
    <p>
      3. Most mathematical symbols weren't invented until the 16th century. Before that, equations were written in words.
    </p>
    <p>
      4. "Forty" is the only number that is spelt with letters arranged in alphabetical order.
    </p>
    <p>
      5. Conversely, "one" is the only number that is spelt with letters arranged in descending order.
    </p>
    <p>
      6. From 0 to 1000, the only number that has the letter "a" in it is "one thousand".
    </p>
    <p>
      7. 'Four' is the only number in the English language that is spelt with the same number of letters as the number itself.
    </p>
    <p>
      8. Every odd number has an "e" in it.
    </p>
    <p>
      9.Over 2000 years ago, Eratosthenes estimated the Earth's circumference using math, without ever leaving Egypt, and he was accurate to within 2%
    </p>
    <p>
      10. Markings on animal bones indicate that humans have been doing math since around 30,000BC.
    </p>
    <p>
      11. "Eleven plus two" is an anagram of "twelve plus one" which is pretty fitting as the answer to both equations is 13.
    </p>
    <p>
      12. Also, there are 13 letters in both "eleven plus two" and "twelve plus one".
    </p>
    <p>
      13. Zero is not represented in Roman numerals.
    </p>
    <p>
      14. The word "mathematics" only appears in one Shakespearean play, "The Taming of the Shrew".
    </p>
    <p>
      15. -40 °C is equal to -40 °F.
    </p>
    <p>
      16. In France, a pie chart is sometimes referred to as a "camembert".
    </p>
    <p>
      17. The symbol for division (i.e.÷) is called an obelus.
    </p>
    <p>
      18. 2 and 5 are the only prime numbers that end in 2 or 5.
    </p>
    <p>
      19. A 'jiffy' is an actual unit of time. It means 1/100th of a second.
    </p>
    <p>
      20. If you shuffle a deck of cards properly, it's more than likely that the exact order of the cards you get has never been seen before in the whole history of the universe.
    </p>
  </div>
</div>
`;

exports[`Components Snapshots renders correctly CalContent 1`] = `
<div
  id="calContent"
>
  Lets Do some math !
</div>
`;

exports[`Components Snapshots renders correctly Calculator 1`] = `
<div
  id="calContainer"
>
  <div
    id="calScreen"
  >
    0
  </div>
  <div
    id="calKeys"
  >
    <button
      className="calKey acKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      AC
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      +/-
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      %
    </button>
    <button
      className="calKey opColor"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      ÷
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      7
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      8
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      9
    </button>
    <button
      className="calKey opColor"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      x
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      4
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      5
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      6
    </button>
    <button
      className="calKey opColor"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      -
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      1
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      2
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      3
    </button>
    <button
      className="calKey opColor"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      +
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      0
    </button>
    <button
      className="calKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      .
    </button>
    <button
      className="calKey eqKey"
      onClick={[Function]}
      onKeyPress={[Function]}
      type="button"
    >
      =
    </button>
  </div>
</div>
`;

exports[`Components Snapshots renders correctly CalculatorPage 1`] = `
<div
  id="calPage"
>
  <div
    id="calContent"
  >
    Lets Do some math !
  </div>
  <div
    id="calContainer"
  >
    <div
      id="calScreen"
    >
      0
    </div>
    <div
      id="calKeys"
    >
      <button
        className="calKey acKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        AC
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        +/-
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        %
      </button>
      <button
        className="calKey opColor"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        ÷
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        7
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        8
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        9
      </button>
      <button
        className="calKey opColor"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        x
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        4
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        5
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        6
      </button>
      <button
        className="calKey opColor"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        1
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        2
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        3
      </button>
      <button
        className="calKey opColor"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        +
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        0
      </button>
      <button
        className="calKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        .
      </button>
      <button
        className="calKey eqKey"
        onClick={[Function]}
        onKeyPress={[Function]}
        type="button"
      >
        =
      </button>
    </div>
  </div>
</div>
`;

exports[`Components Snapshots renders correctly Header 1`] = `
<header
  id="appHeader"
>
  <span
    id="logo"
  >
    LIFE
    <sub>
      OF
    </sub>
    PI
  </span>
  <nav
    id="navbar"
  >
    <ul>
      <li>
        <a
          activeclassname="active"
          className=""
          href="/home"
          onClick={[Function]}
        >
          Home
        </a>
      </li>
      <li>
        <a
          activeclassname="active"
          className=""
          href="/calc"
          onClick={[Function]}
        >
          Calculator
        </a>
      </li>
      <li>
        <a
          activeclassname="active"
          className=""
          href="/quote"
          onClick={[Function]}
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</header>
`;

exports[`Components Snapshots renders correctly HomePage 1`] = `
<div
  id="homePage"
>
  <p>
    if you were to play a word association game with a school-aged child, you'd be pretty unlikely to get a response of 'cool' when you asked them what word first came into their mind when you said 'math'. Despite what some people may tell you, math is far from dull. On the contrary, there are plenty of fun and strange math-related facts out there that will fascinate children of all ages. To prove this, we've compiled a list of 20 cool facts about math which we encourage you to share with the children in your life.
  </p>
  <p>
    1. The word "hundred" comes from the old Norse term, "hundrath", which actually means 120 and not 100.
  </p>
  <p>
    2. In a room of 23 people there's a 50% chance that two people have the same birthday.
  </p>
  <p>
    3. Most mathematical symbols weren't invented until the 16th century. Before that, equations were written in words.
  </p>
  <p>
    4. "Forty" is the only number that is spelt with letters arranged in alphabetical order.
  </p>
  <p>
    5. Conversely, "one" is the only number that is spelt with letters arranged in descending order.
  </p>
  <p>
    6. From 0 to 1000, the only number that has the letter "a" in it is "one thousand".
  </p>
  <p>
    7. 'Four' is the only number in the English language that is spelt with the same number of letters as the number itself.
  </p>
  <p>
    8. Every odd number has an "e" in it.
  </p>
  <p>
    9.Over 2000 years ago, Eratosthenes estimated the Earth's circumference using math, without ever leaving Egypt, and he was accurate to within 2%
  </p>
  <p>
    10. Markings on animal bones indicate that humans have been doing math since around 30,000BC.
  </p>
  <p>
    11. "Eleven plus two" is an anagram of "twelve plus one" which is pretty fitting as the answer to both equations is 13.
  </p>
  <p>
    12. Also, there are 13 letters in both "eleven plus two" and "twelve plus one".
  </p>
  <p>
    13. Zero is not represented in Roman numerals.
  </p>
  <p>
    14. The word "mathematics" only appears in one Shakespearean play, "The Taming of the Shrew".
  </p>
  <p>
    15. -40 °C is equal to -40 °F.
  </p>
  <p>
    16. In France, a pie chart is sometimes referred to as a "camembert".
  </p>
  <p>
    17. The symbol for division (i.e.÷) is called an obelus.
  </p>
  <p>
    18. 2 and 5 are the only prime numbers that end in 2 or 5.
  </p>
  <p>
    19. A 'jiffy' is an actual unit of time. It means 1/100th of a second.
  </p>
  <p>
    20. If you shuffle a deck of cards properly, it's more than likely that the exact order of the cards you get has never been seen before in the whole history of the universe.
  </p>
</div>
`;

exports[`Components Snapshots renders correctly Key 1`] = `
<button
  className="calKey"
  onClick={[Function]}
  onKeyPress={[Function]}
  type="button"
>
  0
</button>
`;

exports[`Components Snapshots renders correctly Keyboard 1`] = `
<div
  id="calKeys"
>
  <button
    className="calKey acKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    AC
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    +/-
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    %
  </button>
  <button
    className="calKey opColor"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    ÷
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    7
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    8
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    9
  </button>
  <button
    className="calKey opColor"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    x
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    4
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    5
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    6
  </button>
  <button
    className="calKey opColor"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    -
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    1
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    2
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    3
  </button>
  <button
    className="calKey opColor"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    +
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    0
  </button>
  <button
    className="calKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    .
  </button>
  <button
    className="calKey eqKey"
    onClick={[Function]}
    onKeyPress={[Function]}
    type="button"
  >
    =
  </button>
</div>
`;

exports[`Components Snapshots renders correctly Navbar 1`] = `
<nav
  id="navbar"
>
  <ul>
    <li>
      <a
        activeclassname="active"
        className=""
        href="/home"
        onClick={[Function]}
      >
        Home
      </a>
    </li>
    <li>
      <a
        activeclassname="active"
        className=""
        href="/calc"
        onClick={[Function]}
      >
        Calculator
      </a>
    </li>
    <li>
      <a
        activeclassname="active"
        className=""
        href="/quote"
        onClick={[Function]}
      >
        Quote
      </a>
    </li>
  </ul>
</nav>
`;

exports[`Components Snapshots renders correctly QuotePage 1`] = `
<div
  id="quotePage"
>
  <blockquote
    cite="http://www.worldwildlife.org/who/index.html"
  >
    Mathematics is the music of reason.
  </blockquote>
  <p>
    -- James Joseph Sylvester, English mathematician:
  </p>
</div>
`;

exports[`Components Snapshots renders correctly Screen 1`] = `
<div
  id="calScreen"
>
  0
</div>
`;
